tracer/src/Datadog.Trace/Tagging/TraceTagCollection.cs

```diff
@@ -262,7 +262,7 @@ public void FixTraceIdTag(TraceId traceId)
                 // (parse the hex string and compare ulongs to avoid allocating another string)
                 if (tagValue == null || !HexString.TryParseUInt64(tagValue, out var currentValue) || currentValue != traceId.Upper)
                 {
-                    SetTag(Trace.Tags.Propagated.TraceIdUpper, HexString.ToHexString(traceId.Upper));
+                    SetTag(Trace.Tags.Propagated.TraceIdUpper, traceId.ToUpperHexString());
```

Comment:
I'm not a fan of this at all 😛  `traceId.ToUpperHexString()` and `ToHexString(traceId.Upper)` mean very different things to me. The former means "convert `traceId` to an uppercase hex string". The latter means "convert the upper bytes to a hex string".

Also, the `ToUpperHexString()` methods seems to do exactly the same as `HexString.ToHexString(Upper)` so I don't see why it's necessary 😉 

```suggestion
                    SetTag(Trace.Tags.Propagated.TraceIdUpper, HexString.ToHexString(traceId.Upper));
```

Reply:
It does the same yes, it's a copy paste. I like to have the logic of what we want to do with those traceids in the trace id function. But I don't mind reverting if you' re really against (I mean my prio is to deliver, not refacto :)) 
I understand the name could be better. `UpperToHexString` would have been better. 
So no go and I revert? or `UpperToHexString` is ok?

Reply:
I mean, I'd prefer to revert, but see how others feel 😉

Reply:
Done in 5879a4e6f33dc6e8fc21d3bf2e03cdf01eeb20d1